<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>711</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>290</y>
       <width>281</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>시작하기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="review_4">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>570</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="shop_name_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>130</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>검색한쇼핑몰</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFourth">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>440</y>
       <width>201</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>신세계.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="price_1">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="shop_name_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>검색한쇼핑몰</string>
     </property>
    </widget>
    <widget class="QLabel" name="price_2">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>260</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="name_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>450</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="shop_name_4">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>400</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>검색한쇼핑몰</string>
     </property>
    </widget>
    <widget class="QLabel" name="price_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>530</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="name_1">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>180</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="review_2">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>300</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="review_1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>300</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="name_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>180</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="review_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>570</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="shop_name_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>검색한쇼핑몰</string>
     </property>
    </widget>
    <widget class="QPushButton" name="next_btn">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>670</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelThird">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>201</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>옥션.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="prev_btn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>670</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>prev</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>의류 비교 다나와</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelFirst">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>201</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>11번가.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="price_4">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>530</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="name_4">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>450</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>상의</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>하의</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>아우터</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelSecond">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>170</y>
       <width>201</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>지마켓.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="search_btn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>조회하기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_top">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>670</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>rank</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_link_1">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>350</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>구매링크</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_link_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>350</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>구매링크</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_link_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>620</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>구매링크</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_link_4">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>620</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>구매링크</string>
     </property>
    </widget>
    <widget class="QLabel" name="price_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>가격:</string>
     </property>
    </widget>
    <widget class="QLabel" name="review_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>300</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>리뷰 수:</string>
     </property>
    </widget>
    <widget class="QLabel" name="price_6">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>260</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>가격:</string>
     </property>
    </widget>
    <widget class="QLabel" name="review_6">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>300</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>리뷰 수:</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>상품명:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="name_6">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>160</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>상품명:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="name_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>430</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>상품명:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="name_8">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>430</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>상품명:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="price_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>530</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>가격:</string>
     </property>
    </widget>
    <widget class="QLabel" name="price_8">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>530</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>가격:</string>
     </property>
    </widget>
    <widget class="QLabel" name="review_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>570</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>리뷰 수:</string>
     </property>
    </widget>
    <widget class="QLabel" name="review_8">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>570</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>리뷰 수:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_graph">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>그래프.png</normaloff>그래프.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>35</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
